python/flexi_cfg_py.cpp

```diff
@@ -179,7 +180,20 @@ PYBIND11_MODULE(flexi_cfg, m) {
                                                                       pyFlexiCfgException);
 
   py::class_<flexi_cfg::Reader>(m, "Reader")
-      .def("dump", &flexi_cfg::Reader::dump)
+      .def(
+          "json",
+          [](const flexi_cfg::Reader& r, bool pretty) -> std::string {
+            if (pretty) {
+              auto json_visitor = flexi_cfg::visitor::PrettyJsonVisitor();
+              r.visit(json_visitor);
+              return json_visitor;
+            }
+            auto json_visitor = flexi_cfg::visitor::JsonVisitor();
+            r.visit(json_visitor);
+            return json_visitor;
+          },
+          py::arg("pretty")=true)
```

Comment:
Maybe this is super nitpicky but should we default to not pretty parsing it?

1. The python script that calls this won't care if it's human-readable or not. Pretty parsing  inflates the json string length unnecessarily.
2. There are already standardized ways to pretty print json objects in Python such as `json.dumps(parsed, indent=4)` and `pprint.pprint(json_object)`. Even `jq` pretty-prints by default I think.